### PR TITLE
[@types/ts-nameof] Add nameof.split

### PR DESCRIPTION
**Pull request by dsherret**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/dsherret/ts-nameof/issues/21#issuecomment-445117492
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,5 +31,6 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
-    }
+    },
+    "dependencies": {}
```
Comment (sheetalkamat) [inline package.json]: Please revert this change in package.json

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dsherret) [reply package.json]: Oops! npm or yarn (forget what I ran) must have automatically done this when re-running install and test before committing. This is reverted now.
